notebooks/Benchmark_notebook.ipynb

```diff
@@ -147,35 +164,38 @@
     "    \n",
     "    return cubes\n",
     "\n",
-    "def quadpy_trapezoid_3d(fn,dim,N,domain):\n",
+    "def quadpy_trapezoid_3d(test_fn,dim,N,domain):\n",
     "    \"\"\"3D trapezoid using quadpy, note that it will perform 8*N evals\n",
     "    \"\"\"\n",
+    "    start_grid = perf_counter()\n",
     "    cubes = _quadpy_helper(N,domain)\n",
     "#     plot_cubes(cubes)\n",
+    "    runtime_grid = perf_counter() - start_grid\n",
     "    scheme = quadpy.c3.product(quadpy.c1.newton_cotes_closed(1))\n",
-    "    val = scheme.integrate(fn,cubes)\n",
-    "    return 0.5*sum(val) #this 0.5 is very odd because it implies that we integrate each domain twice, however we don't?\n",
+    "    return 0.5 * sum(scheme.integrate(test_fn,cubes)), runtime_grid, runtime_f #this 0.5 is very odd because it implies that we integrate each domain twice, however we don't?\n",
```

Comment:
why was this merged into one line? it is now too long to be read

Reply:
```python
val = scheme.integrate(fn,cubes) #this 0.5 is very odd because it implies that we integrate each domain twice, however we don't?
 return val, runtime_grid, runtime_f 
```